<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieDetail</class>
 <widget class="QWidget" name="MovieDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
 color: silver;
 background-image:url(:/icons/texture.png);
 background-clip: border;
 background-color: qlineargradient(spread:reflect,
         x1:1, y1:0, x2:1, y2:1, stop:0 
        rgba(24, 25, 26, 210), stop:1 
        rgba(24, 25, 26, 250));
};</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="upperWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignVCenter">
          <widget class="RemotePixmapLabel" name="poster">
           <property name="minimumSize">
            <size>
             <width>168</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="title">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background:transparent;</string>
               </property>
               <property name="text">
                <string>Title</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="controlButton" name="imdb">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Open IMDB page for this title</string>
               </property>
               <property name="styleSheet">
                <string notr="true">border:none;
background: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/others/imdb-logo.png</normaloff>:/icons/others/imdb-logo.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="controlButton" name="search">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Search this title on Google</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/search-2-line.png</normaloff>:/icons/search-2-line.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="controlButton" name="close">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Close title detail view and go back</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/close-fill.png</normaloff>:/icons/close-fill.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="styleSheet">
              <string notr="true">QScrollArea{
	border:none;
}

QScrollBar:vertical {
	   background-color: transparent;
	   border:none;
       width: 10px;
       margin: 22px 0 22px 0;
   }
   QScrollBar::handle:vertical {
       background: grey;
       min-height: 20px;
   }</string>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>560</width>
                <height>179</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#scrollAreaWidgetContents{
	background: transparent;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_5">
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="tagline">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="font">
                          <font>
                           <pointsize>14</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">background:transparent;</string>
                         </property>
                         <property name="text">
                          <string>tag</string>
                         </property>
                         <property name="wordWrap">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="Line" name="styled_line">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>1</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>1</height>
                          </size>
                         </property>
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="meta">
                         <property name="styleSheet">
                          <string notr="true">background:transparent;</string>
                         </property>
                         <property name="text">
                          <string>meta</string>
                         </property>
                         <property name="wordWrap">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="status">
                         <property name="styleSheet">
                          <string notr="true">background:transparent;</string>
                         </property>
                         <property name="text">
                          <string>Status:</string>
                         </property>
                         <property name="wordWrap">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="ratiingLayout"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_6">
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>9</number>
                       </property>
                       <item alignment="Qt::AlignTop">
                        <widget class="QLabel" name="summery">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">background:transparent;</string>
                         </property>
                         <property name="text">
                          <string>-</string>
                         </property>
                         <property name="wordWrap">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="Line" name="styled_line_2">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>1</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>1</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="crew"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="cVerticalLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
          <property name="text">
           <string>Cast</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="results">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="flow">
           <enum>QListView::LeftToRight</enum>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>controlButton</class>
   <extends>QPushButton</extends>
   <header>widgets/controlbutton.h</header>
  </customwidget>
  <customwidget>
   <class>RemotePixmapLabel</class>
   <extends>QLabel</extends>
   <header>remotepixmaplabel.h</header>
  </customwidget>
  <customwidget>
   <class>cVerticalLabel</class>
   <extends>QLabel</extends>
   <header>widgets/cverticallabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
